pyproject.toml

```diff
@@ -1,9 +1,30 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
 [tool.poetry]
 name = "pyhumps"
 version = "3.8.0"
 description = "Convert strings (and dictionary keys) between snake case, camel case and pascal case in Python."
-authors = ["Nick Ficano <nficano@gmail.com>"]
 license = "The Unlicense (Unlicense)"
+authors = ["Nick Ficano <nficano@gmail.com>"]
+readme = "README.md"
+homepage = "https://github.com/nficano/humps"
+repository = "https://github.com/nficano/humps"
+documentation = "https://humps.readthedocs.io/"
+keywords = ["humps", "snakecase", "convert case", "camelcase", "kebabcase"]
+classifiers = [
+    "License :: OSI Approved :: The Unlicense (Unlicense)",
+    "Programming Language :: Python",
```

Comment:
Hello, actually, you don't need to add Python classifiers if you're using poetry publish. Link to poetry documentation https://python-poetry.org/docs/pyproject/#classifiers.

Reply:
Oh thanks for letting me know, I'll update the PR